### ISSUE TITLE
Backend: Discussion

### ISSUE THREAD

**Issue by jnv27** [srrathi/Hire-Smart#2]
Hello @srrathi ,
I would like to work on backend part of this project as part of GSSOC'21. Can you mention the requirements? 
Also, let me know when we have a group for discussion.

**Comment by Vishal1003** [comment]
Yes. Kindly give a blueprint of the req. I would like to contribute as a GSSOC'21 participant.

**Comment by Kashish0423** [comment]
Yes, eagerly waiting to contribute towards the workflow of this amazing idea as a GSSOC'21 participant using Nodejs  and EJS

**Comment by srrathi** [comment]
Hello everyone,
Thank you so much for thinking of to work on this project. I'll be more
than happy to work and discuss about the project with all of you and take
it to its ultimate completion but I'm sorry to say that right now my
semester exams are ongoing if possible can we discuss about it and how we
will go further after 5 days. It'll be very helpful for me to get into it
completely and work on it.

Thank you all of you

**Comment by Sukriti-sood** [comment]
@srrathi  I would also like to work for same.

**Comment by micksharma** [comment]
I want to work on this issue as a GSSOC 2021 participant.

**Comment by rashtron** [comment]
Hey @srrathi,
I would like to contribute towards Blockchain and Ethereum aspects in the above project as a part of GSSOC'21 participant as I'm also start learning about the same concepts and would consider myself as a mere beginner in it but I'm looking forward to learn about various concepts with the help of this project.

Looking forward to an exciting journey with all of you guys!

**Comment by SrirajBehera** [comment]
@srrathi I am also a GSSOC Participant and want to contribute to this project. To be honest, I know only a few things from the tech stack of this project and I am ready to learn and contribute. So can you please guide me on how and where to start?

**Comment by Esha-Sherring** [comment]
@srrathi I am also a GSSOC Participant and want to contribute to this project I am a backend developer as well as a UI/UX designer

**Comment by drishtipeshwani** [comment]
@srrathi I am a GSSOC Participant and would like to contribute on the Front-end part of the project

**Comment by Adrenalinerush07** [comment]
@srrathi I am a GSSOC Participant and would like to contribute to the Blockchain and Ethereum aspects of the project along with the backend part.

**Comment by abhilasha007** [comment]
@srrathi Hey I would like to contribute to this project. I can contribute in frontend development.

**Comment by OnewayShruti** [comment]
Hey @srrathi
I am a GSSOC'21 participant(Discord Username -Oneway_Shruti(P)#9537)
I wish to contribute to Hire-Smart project.
I am familiar with Ethereum blockchain ,also developed smart contracts for CRUD operations and storing data in a decentralized manner using the IPFS file system.
Please consider assigning me this issue. I assure you ,I will handle the issue well.
Also please forgive me I created an issue instead of commenting .Apologies.

**Comment by ahk4815** [comment]
Hey @srrathi
I am a GSSOC'21 participant(Discord Username -Ali(P)#9323)
I wish to contribute to Hire-Smart project.

**Comment by Spirit-ofJoy** [comment]
@srrathi I am also interested in contributing to the project. Am a GSSOC '21 participant and would love to contribute at the frontend.

**Comment by ahk4815** [comment]
@srrathi I am interested and keen on working on this project.

**Comment by udaymittal7** [comment]
@srrathi I am interested in contributing to the backend for this. I am a participant of GSSOC'21.

**Comment by abhay27chauhan** [comment]
@srrathi I am interested in contributing to the backend for this. I am a participant of GSSOC'21.

**Comment by simran2607** [comment]
Hey would like to contribute. pls provide the requirements.

**Comment by iamharsh1312** [comment]
@srrathi  I would like to contribute to this project. I am a participant of GSSOC'21
